src/codechicken/nei/NEIClientConfig.java

```diff
@@ -7,23 +7,9 @@
 import codechicken.lib.config.ConfigFile;
 import codechicken.lib.config.ConfigTag;
 import codechicken.lib.config.ConfigTagParent;
-import codechicken.nei.api.API;
-import codechicken.nei.api.GuiInfo;
-import codechicken.nei.api.IConfigureNEI;
-import codechicken.nei.api.ItemInfo;
-import codechicken.nei.api.NEIInfo;
-import codechicken.nei.config.ConfigSet;
-import codechicken.nei.config.GuiHighlightTips;
-import codechicken.nei.config.GuiNEIOptionList;
-import codechicken.nei.config.GuiOptionList;
-import codechicken.nei.config.OptionCycled;
-import codechicken.nei.config.OptionGamemodes;
-import codechicken.nei.config.OptionList;
-import codechicken.nei.config.OptionOpenGui;
-import codechicken.nei.config.OptionTextField;
-import codechicken.nei.config.OptionToggleButton;
-import codechicken.nei.config.OptionToggleButtonBoubs;
-import codechicken.nei.config.OptionUtilities;
+import codechicken.nei.api.*;
```

Comment:
Please revert the import changes.  I've set my IDE to _not_ collapse imports and prefer explicit imports.